src/boards/include/boards/sparkfun_micromod.h

```diff
@@ -0,0 +1,64 @@
+/*
+ * SPDX-License-Identifier: BSD-3-Clause
```

Comment:
I'm obviously not a legal-expert :wink: but does the [BSD-3-Clause](https://opensource.org/licenses/BSD-3-Clause) _require_ a `<YEAR>` and `<COPYRIGHT HOLDER>`? Or is it safe to assume those from the owner-of-the-repo and the date-of-commit?

Reply:
fixed